app/src/main/java/com/keylesspalace/tusky/db/InstanceEntity.kt

```diff
@@ -25,8 +25,9 @@ import com.keylesspalace.tusky.entity.Emoji
 
 @Entity
 @TypeConverters(Converters::class)
-data class EmojiListEntity(@field:PrimaryKey var instance: String,
-                           val emojiList: List<Emoji>)
+data class InstanceEntity(@field:PrimaryKey var instance: String,
```

Comment:
Please format it like you formatted `Instance` with each field on its own like thx :heart: